pkg/exchange/bitget/exchange.go

```diff
@@ -182,9 +184,105 @@ func (e *Exchange) QueryAccountBalances(ctx context.Context) (types.BalanceMap,
 	return bals, nil
 }
 
+// SubmitOrder submits an order.
+//
+// Remark:
+// 1. We support only GTC for time-in-force, because the response from queryOrder does not include time-in-force information.
+// 2. For market buy orders, the size unit is quote currency, whereas the unit for order.Quantity is in base currency.
+// Therefore, we need to calculate the equivalent quote currency amount based on the ticker data.
 func (e *Exchange) SubmitOrder(ctx context.Context, order types.SubmitOrder) (createdOrder *types.Order, err error) {
-	// TODO implement me
-	panic("implement me")
+	if len(order.Market.Symbol) == 0 {
+		return nil, fmt.Errorf("order.Market.Symbol is required: %+v", order)
+	}
+
+	req := e.v2Client.NewPlaceOrderRequest()
+	req.Symbol(order.Market.Symbol)
+
+	// set order type
+	orderType, err := toLocalOrderType(order.Type)
+	if err != nil {
+		return nil, err
+	}
+	req.OrderType(orderType)
+
+	// set side
+	side, err := toLocalSide(order.Side)
+	if err != nil {
+		return nil, err
+	}
+	req.Side(side)
+
+	// set quantity
+	orderQty := order.Quantity
```

Comment:
since we're in the order method, this could be named as just "qty" or "q", "quantity", the prefix "order" is not necessary